src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.service.impl.DataConverterImpl;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.ShopServiceImpl;
+import core.basesyntax.service.impl.operation.BalanceOperation;
+import core.basesyntax.service.impl.operation.PurchaseOperation;
+import core.basesyntax.service.impl.operation.ReturnOperation;
+import core.basesyntax.service.impl.operation.SupplyOperation;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String PATH_TO_REPORT_FILES = "src/main/resources/";
```

Comment:
Avoid hardcoding file paths. It's better to receive them as arguments to the main method or from a configuration file.

Reply:
++